forge-gui/res/editions/Edge of Eternities Stellar Sights.txt

```diff
@@ -1,12 +1,188 @@
 [metadata]
 Code=EOS
 Date=2025-08-01
-Name=Edge of Eternities: Stellar Sights
+Name=Edge of Eternities: Steller Sights
 Type=Collector_Edition
 ScryfallCode=EOS
 
 [cards]
+1 M Ancient Tomb @Bruce Brenneise
```

Comment:
Looks like you're not working with the latest files here? The editions already have all cards in the latest updates